lib/dpdkdevice.cc

```diff
@@ -416,6 +416,26 @@ int DPDKDevice::initialize_device(ErrorHandler *errh)
         }
     }
 
+    if (info.init_fc_mode != FC_UNSET) {
+        struct rte_eth_fc_conf conf;
+        ret = rte_eth_dev_flow_ctrl_get(port_id, &conf);
+        if (ret != 0)
+            return errh->error("Could not get flow control status !");
+        switch (info.init_fc_mode) {
```

Comment:
[Detail] You could allocate a static Map with string as key and the enum as a value.
this way you can simply get from that map